### PR TITLE
Files for example added

### PR DESCRIPTION
**Pull request by kmith**
Modified "diffusion_equation" example conforming to new format.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+  cd ./diffusion_equation/src/python
+  source  /path/to/opencmisslibs/install/.../.../virtualenvironments/oclibs_pyXY_release/bin/activate
+  python diffusion_equation.py
+  
+  Note: If the above fails, try the following.
+  cd ./diffusion_equation/src/python
+  PYTHONPATH="/path/to/opencmisslibs/install/.../.../Release/opencmiss.iron" python diffusion_equation.py  
+
+Results can be visualised by running `visualise.com <./src/fortran/visualise.com>`_ with the `Cmgui visualiser <http://physiomeproject.org/software/opencmiss/cmgui/download>`_.
```
Comment (hsorby) [inline README.rst]: Could make this into a  [verify section](https://github.com/OpenCMISS-Examples/template_example/blob/master/README.template.rst#verifying-the-example) and add a comment on how the expected results are available.

**Review by hsorby — CHANGES_REQUESTED** [review]
Looks good creating a verify section would make this up-to-date with the current template and we could merge it in then.

**Comment by chrispbradley** [comment]
And maybe change visualize.com to visualize.cmgui???

**Review by hsorby — APPROVED** [review]
Great looks good.

**Pull request merged**